doped/utils/parsing.py

```
@@ -1988,3 +2009,374 @@ def _multiple_files_warning(file_type, directory, chosen_filepath, action=None,
         f"Multiple `{file_type}` files found in {dir_type} directory: {directory}. Using {filename} to "
         f"{action}"
     )
+
+
+
+from pymatgen.entries.computed_entries import ComputedEntry
```

Ruff (0.12.2)

Module level import not at top of file

(E402)